daft/dataframe/dataframe.py

```diff
@@ -402,11 +402,25 @@ def write_parquet(
         write_df.collect()
         assert write_df._result is not None
 
-        # Populate and return a new disconnected DataFrame
-        result_df = DataFrame(write_df._builder)
-        result_df._result_cache = write_df._result_cache
-        result_df._preview = write_df._preview
-        return result_df
+        if len(write_df) > 0:
+            # Populate and return a new disconnected DataFrame
+            result_df = DataFrame(write_df._builder)
+            result_df._result_cache = write_df._result_cache
+            result_df._preview = write_df._preview
+            return result_df
+        else:
+            from daft import from_pydict
+            from daft.table.table_io import write_empty_tabular
+
+            file_path = write_empty_tabular(
+                root_dir, FileFormat.Parquet, self.schema(), compression=compression, io_config=io_config
+            )
+
+            return from_pydict(
+                {
+                    "path": [file_path],
```

Comment:
Do we only return the `path` in result_df, or is there more? I think we might have things like size and num_rows as well

Reply:
We only return the path right now